src/utils/stockDataUtils/calculations.js

```diff
@@ -0,0 +1,14 @@
+import { parseIntExact } from '../typeChecking';
+
+// returns a decimal, multiply by 100 for a percentage
```

Comment:
I wonder if we should use `/** */` comments on exported functions instead of `//` comments.  It is the JSDoc way: http://usejsdoc.org/about-getting-started.html